### PR TITLE
enh: check if the directory exists before installing on mac

### PR DESCRIPTION
**Pull request by facetosea**

### CODE REVIEW

File: packaging/tools/post.sh [hunk 1]
```diff
@@ -627,9 +627,16 @@ function install_app() {
   fi
 }
 
-function install_TDengine() {
-    echo -e "${GREEN}Start to install TDengine...${NC}"
-    log_print "start to install TDengine"
+function checkDirectory() {
+    if [ ! -d "/usr/local/bin" ]; then
+      sudo mkdir -p /usr/local/bin
```
Comment (sangshuduo) [inline packaging/tools/post.sh]: need consider an environment like docker has no sudo command.

**Review by sangshuduo — CHANGES_REQUESTED** [review]
see my comments

File: packaging/tools/post.sh (hunk 1 above)
Comment (facetosea) [reply packaging/tools/post.sh]: Adapted to different platforms

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**